gui/slick/views/layouts/main.mako

```diff
@@ -117,6 +117,12 @@
                                 <li><a href="${srRoot}/home/"><i class="menu-icon-home"></i>&nbsp;Show List</a></li>
                                 <li><a href="${srRoot}/home/addShows/"><i class="menu-icon-addshow"></i>&nbsp;Add Shows</a></li>
                                 <li><a href="${srRoot}/home/postprocess/"><i class="menu-icon-postprocess"></i>&nbsp;Manual Post-Processing</a></li>
+                                % if sickbeard.SHOWS_RECENT:
```

Comment:
Check if `SHOWS_RECENT` is not empty? And I think that `RECENT_SHOWS` is a better name.